api/src/Service/DateTimeProvider.php

```diff
@@ -2,10 +2,11 @@
 
 namespace App\Service;
 
-class DateTimeProvider
+class DateTimeProvider extends \App\Service\Time\DateTimeProvider
```

Comment:
I don't think we need this here - we have two classes named `DateTimeProvider` in API and Client so trying to extend one from the other will break our code as the classes aren't available to each other. Was there a reason you needed to do this?

Reply:
Sorry I think I did this in error,  I'll revert this back.